tests/acceptance/README

```diff
@@ -258,3 +258,41 @@ Soft fail: the test failed as expected by a "test_soft_fail" promise.
 
 Skipped: test is skipped due to be either explicitly disabled or being
 Nova-specific and being run on Community cf-agent.
+
+------------------------------------------------------------------------------
+Example Test Skeleton
+------------------------------------------------------------------------------
+body common control
+{
+      inputs => { "../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+# Initalize the environment and prepare for test
+{
+
+}
+
+bundle agent test
+# Activate policy for behaviour you wish to inspect
+{
+  meta:
+    "description" string => "What does this test?";
+    "story_id" string => "5525";
+    "covers" string => "operational_repaired";
+
+    "test_soft_fail"
```

Comment:
come comment on what this does, also another entry on how to skip a platform would also be useful.